pkg/virt-api/webhooks/validating-webhook/admitters/instancetype-admitter_test.go

```diff
@@ -47,6 +47,27 @@ var _ = Describe("Validating Instancetype Admitter", func() {
 		Expect(response.Allowed).To(BeFalse(), "Expected instancetype to not be allowed")
 		Expect(response.Result.Code).To(Equal(int32(http.StatusBadRequest)), "Expected error 400: BadRequest")
 	})
+
+	It("should reject instanceType with dedicatedCPUPlacement", func() {
+		instancetypeObj = &instancetypev1alpha2.VirtualMachineInstancetype{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "test-name",
+				Namespace: "test-namespace",
+			},
```

Comment:
`BeforeEach` already provides a basic InstancetypeObj with this set, all you need to do here is update the spec.

```
instancetypeObj.Spec = instancetypev1alpha2.VirtualMachineInstancetypeSpec{
	CPU: instancetypev1alpha2.CPUInstancetype{
		DedicatedCPUPlacement: true,
	},
}
```